data/tlg0555/tlg001/tlg0555.tlg001.opp-grc1.xml

```diff
@@ -24,6 +24,10 @@
                 <pubPlace>Leipzig</pubPlace>
                 <date>1905</date></imprint>
             </monogr>
+           <series>
```

Comment:
`<title>Protrepticus und Paedagogus</title>` (above) should probably have a "mul" language attribute